doc/api/tls.md

```diff
@@ -7,7 +7,7 @@
 <!-- source_link=lib/tls.js -->
 
 The `node:tls` module provides an implementation of the Transport Layer Security
-(TLS) and Secure Socket Layer (SSL) protocols that is built on top of OpenSSL.
+(TLS) and Secure Socket Layer (SSL) protocols that are built on top of OpenSSL.
```

Comment:
There is no typo here. The 'is' here is referring to the `node:tls` module, not 'protocols'.